autoware_map_msgs/srv/DifferentialPointCloudMap.srv

```diff
@@ -0,0 +1,13 @@
+# Area query for map loading
+# (PCD grids that overlaps with the area is going to be loaded)
+AreaInfo area
+
+# The IDs of PCD maps that the node already has
+string[] already_loaded_ids
```

Comment:
I would like to get from native speakers regarding naming.
cc @mitsudome-r
example:
- hold_ids
- loaded_ids
- cached_ids

Reply:
@mitsudome-r Would you mind giving us some comments on these reviews? (Mainly about the variable names)

Reply:
Continued in https://github.com/autowarefoundation/autoware_msgs/pull/39#discussion_r991881747